### PR TITLE
feat-tracker

### PR DESCRIPTION
**Pull request by Harrikar**

### CODE REVIEW

File: EarthMC/Classes/Nations.py [hunk 1]
```diff
@@ -18,9 +19,12 @@ def __init__(self, name="", king="", capital="", residents=None, towns=None, are
         self.towns = towns
         self.area = area
 
+
 class nations:
+
+
     def __init__(self, mapName):
-        self.Towns = towns(mapName)
+        self.Towns = Town(mapName)
```
Comment (Owen3H) [inline EarthMC/Classes/Nations.py:27]: this is probably a mistake.

File: EarthMC/DataHandler.py
Comment (Owen3H) [inline EarthMC/DataHandler.py:1]: File doesn't need to be touched.

File: EarthMC/Classes/OAPI/Nation.py [hunk 2]
```diff
@@ -21,6 +21,8 @@ def __init__(self, data):
         self.area = stats['numTownBlocks']
         self.residents = data['residents']
         self.towns = data['towns']
+        self.open = data['isOpen']
```
Comment (Owen3H) [inline EarthMC/Classes/OAPI/Nation.py:24]: These are only accessible on `data['status']`

**Comment by Harrikar** [comment]
is it good now

File: EarthMC/Classes/Nations.py (hunk 1 above)
Comment (Harrikar) [reply EarthMC/Classes/Nations.py:27]: yea

**Comment by Owen3H** [comment]
Better, but you are still initializing and using `aurora_map` and `nova_map` inany places.
This project is built on inheritance so I don't wanna deviate from that and make it a mess.

**Comment by Harrikar** [comment]
i will fix it i just got home from the barber

**Comment by Owen3H** [comment]
Ty. Essentially, Map is an instance of either aurora or nova. so you should be doing
```py
def __init__(self, map):
    self.map = map

def example_func():
    towns = self.map.Towns.all()
```

**Comment by Harrikar** [comment]
yea imma fix it

**Comment by Harrikar** [comment]
the players though arent on class map

**Comment by Harrikar** [comment]
i believe it is good now

**Pull request merged**